File: ClientApp/ClientApp/Models/PersonalStatistics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientApp.Models
{
    public class PersonalStatistics
    {
        public string UserName {  get; set; }
        public double AvgAnswerTime { get; set; }
        public int GamesPlayed {  get; set; }
        public int TotalAnswers { get; set; }
        public int TotalCorrectAnswers { get; set; }

    }
}

File: M_Trivia/M_Trivia/PersonalStatistics.cpp
#include "PersonalStatistics.hpp"

PersonalStatistics::PersonalStatistics(const std::string& user, int totalCorrect, int games, int totalAns, double avgTime)
    : username(user), totalCorrectAnswers(totalCorrect), gamesPlayed(games),
    totalAnswers(totalAns), avgAnswerTime(avgTime) {
}



